<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPEFileSelectorForm</class>
 <widget class="QWidget" name="HPEFileSelectorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileSelector</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	padding: 2px 8px;
     color: gray;
     background-color: white;
     border-style: solid;
     border-width: 5px;
     border-radius: 8px;
     border-color: rgb(208, 208, 208);
     font: bold 15px;
}

QPushButton:hover
{
	/*border-color: gray;*/
     color: rgb(255, 255, 255);
     background-color: rgb(208, 208, 208);
}

QPushButton:disabled
{
	color: rgb(235, 235, 235);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="backButtonContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentDirLabel">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>Current Dir</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="fileListWidget">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
	border: 4px solid gray; 
	border-radius: 8px; 
	margin: 0px 20px;
	color: gray;
}

QListWidget::item:selected {
	background-color: rgb(208, 208, 208);
	color: white;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="confirmButtonContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
